sunpy/map/sources/tests/test_lasco_source.py

```diff
@@ -14,6 +15,10 @@
 fitspath = glob.glob(os.path.join(path, "lasco_c2_25299383_s.fts"))
 lasco = Map(fitspath)
 
+jp2path = glob.glob(os.path.join(path, "2013_05_13__16_54_06_137__SOHO_LASCO_C3_white-light.jp2"))
+lasco_helioviewer = Map(jp2path)
```

Comment:
I know you are following the style of this file but ideally this should be a pytest fixture like so:
```python
@pytest.fixture()
def lasco_helioviewer():
    return Map(jp2path)
```

Then you can pass "lasco_helioviewer" into the test you wrote.

Reply:
sure, that makes sense. Should I do this only for `lasco_helioviewer` or also adjust the `lasco` file to use a fixture?

Reply:
If you can that would be great.

Reply:
done in 8a458bc.

Reply:
Thanks.